OpenBudgeteer.Blazor/Shared/BucketStatsElement.razor

```diff
@@ -3,17 +3,31 @@
     <MudPaper Class="no-background" Elevation="0">
         <MudText Align="Align.Center" Style="font-weight: bold">@Title</MudText>
         <MudDivider Class="my-1"/>
-        <MudText Align="Align.Center">@Amount</MudText>
+        <MudText Align="Align.Center" Style="@GetBalanceColor(Amount, Title)">@Amount</MudText>
     </MudPaper>
 </MudStack>
 
 @code {
     [Parameter] 
     public string? ImageUrl { get; set; }
-    
+
     [Parameter] 
     public string? Title { get; set; }
-    
+
     [Parameter] 
     public string? Amount { get; set; }
```

Comment:
I would propose to make this a decimal (so that it doesn't have to be parsed in `GetBalanceColor`) and use `@Amount.ToString("C", CultureInfo.CurrentCulture)"` in the `MudText`